runelite-client/src/main/java/net/runelite/client/plugins/prayer/PrayerConfig.java

```diff
@@ -136,8 +136,20 @@ default boolean hideIfOutOfCombat()
 		name = "Replace orb text with prayer time left",
 		description = "Show time remaining of current prayers in the prayer orb."
 	)
+	default boolean showPrayerBarHelper()
+	{
+		return false;
+	}
+
+	@ConfigItem(
+		position = 10,
+		keyName = "showPrayerBarFlickHelper",
+		name = "Shows click indicator for prayer bar",
+		description = "Prayer bar will display flags when clicked to show tick timing."
+
+	)
 	default boolean replaceOrbText()
 	{
 		return false;
 	}
-}
\ No newline at end of file
+}
```

Comment:
Let's try this again
```suggestion
}

```